README.md

```diff
@@ -1,51 +1,50 @@
+This is the home of the schema for the GA4GH Tool Registry API. The GA4GH Tool Registry API is a standard for listing and describing available tools (both stand-alone, containerized tools as well as workflows in CWL, WDL, Nextflow, Galaxy or Snakemake) in a given registry. This defines a minimal, common API describing tools that we propose for support by multiple tool/workflow registries like [Dockstore](https://www.dockstore.org/), [BioContainers](https://biocontainers.pro), and [Agora](https://github.com/broadinstitute/agora) for the purposes of exchange, indexing, and searching.
 
 This repo uses the [HubFlow](https://datasift.github.io/gitflow/) scheme which is closely based on [GitFlow](https://nvie.com/posts/a-successful-git-branching-model/). In practice, this means that the master branch contains the last production release of the schema whereas the develop branch contains the latest development changes which will end up in the next production release. 
-As of July 2019, this means that the 1.0.0 version is described on master whereas the develop branch contains the 2.0.0-beta.3 version which will evolve into the 2.0.0 production release.
+As of February 2022, this means that the 2.0.1 version is described on master whereas the develop branch contains work which will accumulate and evolve into a 2.1 production release.
 
-Our current proposal is to start with a read-only API due to potentially different views and approaches to registration/security.
+Our current iteration focuses on a read-only API due to potentially different views and approaches to registration/security.
 
-Key features of the current API proposal:
+Key features of the current API:
 
 * read-only API
-* May serve up CWL, WDL, Nextflow, Galaxy or Snakemake to describe a tool or represent a workflow 
-  depending on the tool/workflow submitter
-* ID:  globally unique across systems and also identifies the system that it came from (ex: 123456323@agora.broadinstitute.org )
-
+* Serve up one or more of CWL, WDL, Nextflow, Galaxy or Snakemake to describe a tool or represent a workflow depending on the tool/workflow submitter 
+* Download individual workflow files or the entire workflow as a zip
+* Interrogate the language versions of these workflows
+* Get specific versions of workflows and tools, potentially with immutable versions with checksums on their files
+* ID: globally unique across systems and also identifies the system that it came from (ex: 123456323@agora.broadinstitute.org )
+* Refer to our [data model](https://ga4gh.github.io/tool-registry-service-schemas/DataModel/) for information on how to define TRS URIs if desired
+* provides more structure than a simple un-formatted list of tools but it is also a standard for registries to implement as opposed to a registry implementation itself
 
 Outstanding questions:
```

Comment:
Given the ambiguity of meanings, perhaps better "Open questions" rather than "Outstanding questions"?

We might also add a short intro to the list of questions, like:

> Questions TRS currently does not (comprehensively) address include the following: